src/editors/matl.rs

```diff
@@ -194,6 +194,31 @@ fn remove_duplicates(entries: &mut Vec<MatlEntryData>) {
     });
 }
 
+fn remove_unused_materials(matl_entries: &mut Vec<MatlEntryData>, modl_entries: &Vec<ModlEntryData>) {
+    let mut blacklist = vec![
+        String::from("EyeL"), String::from("EyeL1"), String::from("EyeL2"),
+        String::from("EyeR"), String::from("EyeR1"), String::from("EyeR2"),
+        String::from("EyeLD"), String::from("EyeLD1"), String::from("EyeLD2"),
+        String::from("EyeRD"), String::from("EyeRD1"), String::from("EyeRD2"),
+        String::from("EyeLG"), String::from("EyeLG1"), String::from("EyeLG2"),
+        String::from("EyeRG"), String::from("EyeRG1"), String::from("EyeRG2"),
+        String::from("EyeLL"), String::from("EyeLL1"), String::from("EyeLL2"),
+        String::from("EyeRL"), String::from("EyeRL1"), String::from("EyeRL2"),
+    ];
+
+    let mut visited = Vec::with_capacity(modl_entries.len() + blacklist.len());
+    visited.append(&mut blacklist);
```

Comment:
You can just initialize the visited vector with these entries with `let mut visited = vec![...];` The list is small, so I would just focus on readability and not worry about reallocations when inserting.

Reply:
I would also add a comment explaining why these material names are important.